pkg/operator/ceph/disruption/machinedisruption/reconcile.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package machinedisruption
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/coreos/pkg/capnslog"
+	healthchecking "github.com/openshift/machine-api-operator/pkg/apis/healthchecking/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	cephClient "github.com/rook/rook/pkg/daemon/ceph/client"
+	cephCluster "github.com/rook/rook/pkg/operator/ceph/cluster"
+	"github.com/rook/rook/pkg/operator/ceph/disruption/controllerconfig"
+	"github.com/rook/rook/pkg/operator/ceph/disruption/machinelabel"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	controllerName                  = "machinedisruption-controller"
+	MDBCephClusterNamespaceLabelKey = "rook.io/cephClusterNamespace"
+	MDBCephClusterNameLabelKey      = "rook.io/cephClusterName"
+)
+
+var logger = capnslog.NewPackageLogger("github.com/rook/rook", controllerName)
+
+// MachineDisruptionReconciler reconciles MachineDisruption
+type MachineDisruptionReconciler struct {
+	scheme  *runtime.Scheme
+	client  client.Client
+	context *controllerconfig.Context
+}
+
+// Reconcile is the implementation of reconcile function for MachineDisruptionReconciler
+// which ensures that the machineDisruptionBudget for the rook ceph cluster is in correct state
+func (r *MachineDisruptionReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	logger.Infof("reconciling %s", request.NamespacedName)
```

Comment:
Debug here. I was Info logging in the PDB controllers too, but it turned out to be a bit too noisy.

Reply:
done